src/base-input.js

```diff
@@ -53,10 +53,10 @@ import i18n, {notifyOnLangChange, stopNotifyingOnLangChange} from './i18n.js';
  * @csspart helpText - Use for customizing the style of the helpText element
  * @csspart accentIcon - Use for customizing the style of the accentIcon element (e.g. credit card icon, calendar icon)
  * @csspart iconContainer - Use for customizing the style of the iconContainer (e.g. X icon for clearing input value)
- * @event input - Event fires when the value of an `auro-input` has been changed.
+ * @event input - Event fires when the value of an `auro-input` has been changed. Both updated string and `isValid` boolean are accessible.
```

Comment:
All public attributes are accessible as a basic rule. `isValid` is always available to be checked regardless of if `input` event being fired. This seems redundant to me and muddies the explanation of this event.

Reply:
Got it! I can check that out.